perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -68,6 +68,15 @@ sub base_url { FixMyStreet::Cobrand::UKCouncils::base_url($_[0]) }
 
 sub default_map_zoom { 6 }
 
+sub enter_postcode_text {
+    my ( $self ) = @_;
+    return _("Enter a road name or postcode adjacent to the area you want to report on or the name of the area");
+}
+
+sub example_places {
```

Comment:
You could call the UKCouncils function here and use a cobrand feature for this if that was preferred.

Reply:
The UK Councils text is slightly different as cobrand requested text (wasn't sure whether I should use translation trigger for a cobrand though). It seemed easiest to do the example place in the same place while dealing with the query, but happy to move to feature if preferred

Reply:
Yes, I was only talking about example_places - doesn't really matter, I guess, might as well leave it.